inc/template-functions.php

```diff
@@ -164,6 +164,23 @@ function hale_get_branding_class() {
 	return $style_class;
 }
 
+/**
+ * Get the template which has been selected for the page
+ */
+function hale_get_template_class() {
+	$template = get_page_template_slug();
```

Comment:
This will run on all admin pages. Please make it more specific by checking get_current_screen. Thanks

Reply:
Added